app/views/KnownIssuesParagraph.scala.html

```diff
@@ -0,0 +1,51 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@(gbHealthy: Boolean, xiHealthy: Boolean, gbHealthMessage: String, xiHealthMessage: String)(implicit request: Request[_], messages: Messages)
+
+    @if(!gbHealthy && !xiHealthy){
+        <p class="govuk-body">
+            @messages("service.availability.issues.p1")
+            <b>@xiHealthMessage</b>
+            @messages("service.availability.issues.p2")
+            <b>@gbHealthMessage</b>
+            @messages("service.availability.issues.both.channels")
+
+            <!--TODO: change the hardcoded value-->
+            @messages("service.availability.issues.known") <b>2:10pm.</b>
```

Comment:
Can you remove any TODO comments or code with hard coded stuff in please? It's best to just add it when the feature is ready to go in. That way, if we don't implement the feature for whatever reason, it won't get left in by mistake.

Also, it won't give the client the false impression that the feature is already built.